site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -10,36 +10,38 @@ description: >
 ## Local Development
 
 When the game server is running on Agones, the SDK communicates over TCP to a small
-gRPC server that Agones coordinated to run in a container in the same network 
+gRPC server that Agones coordinated to run in a container in the same network
 namespace as it - usually referred to in Kubernetes terms as a "sidecar".
 
 Therefore, when developing locally, we also need a process for the SDK to connect to!
 
 To do this, we can run the same binary that runs inside Agones, but pass in a flag
 to run it in "local mode". Local mode means that the sidecar binary
-will not try to connect to anything, and will just send log messages to stdout and persist local state in memory so 
+will not try to connect to anything, and will just send log messages to stdout and persist local state in memory so
 that you can see exactly what the SDK in your game server is doing, and can
 confirm everything works.
 
-To do this you will need to download the latest agonessdk-server-{VERSION}.zip from 
+To do this you will need to download the latest `agonessdk-server-{{< release-version >}}.zip` from
 [releases](https://github.com/googleforgames/agones/releases), and unzip it.
```

Comment:
Can we point to the specific release instead of making users find it on this page?

Maybe something like `https://github.com/googleforgames/agones/releases/tag/v{{< release-branch >}}`?